packages/ckeditor5-package-generator/lib/utils/validate-global-name.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license Copyright (c) 2020-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const chalk = require( 'chalk' );
+
+/**
+ * If the global name is not valid, prints the error and exits the process.
+ *
+ * @param {Logger} logger
+ * @param {String|undefined} globalName
+ */
+module.exports = function validateGlobalName( logger, globalName ) {
+	if ( !globalName ) {
+		return;
+	}
```

Comment:
It looks like this condition is always falsy, because `globalName` is always defined.